boa3_test/tests/boa_test.py

```diff
@@ -220,12 +230,24 @@ def compile(self, path: str, root_folder: str = None) -> bytes:
         return result
 
     def compile_and_save(self, path: str, root_folder: str = None, debug: bool = False, log: bool = True,
-                         output_path: str = None, **kwargs) -> Tuple[bytes, Dict[str, Any]]:
+                         output_name: str = None, env: str = None, **kwargs) -> Tuple[bytes, Dict[str, Any]]:
```

Comment:
The same question as the comment before

Reply:
In this case I implemented with kwargs to avoid IDE autocompletes to show the new argument
For test compile_and_save, I didn't want to expose `use_unique_name` anywhere outside the file `boa_test`